.husky/pre-commit

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+
+# Check if user have imagemagick installed
+# Check on Linux, MacOS and Windows
+command -v magick >/dev/null 2>&1 || {
+  echo >&2 "imagemagick is not installed. Attempting to install it..."
+  echo >&2 "This is a one time fix..."
+  echo >&2 "If you are on Windows, please install imagemagick manually."
+  # If Host is linux, install imagemagick
+    if [ "$(uname)" == "Linux" ]; then
+        echo >&2 "Installing imagemagick..."
+        sudo apt-get install imagemagick
```

Comment:
Not platform agnostic
Use imagemagick binary and copy it to ~/.bin and make it executable